.github/workflows/e2e-nvidia-t4-x1.yml

```
@@ -21,10 +21,10 @@ on:
       - main
       - release-*
     paths:
-      - files~=\.py$
-      - files=pyproject.toml
-      - files=^requirements.*\.txt$
-      - files=.github/workflows/e2e-nvidia-t4-x1.yml # This workflow
+      - **.py
```

```suggestion
      - "**.py"
```
Need to quote it. Yaml treats leading `*` special.

---

good catch, @bjhargrave